logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/mapper/CallMapper.kt

```diff
@@ -230,7 +230,9 @@ class CallMapperImpl(
                 }
             }
 
-        return MessageTarget.Client(
+        // TODO(MO): this option is set to IgnoreIfMissing just so i do not change how it was,
+        //  but it should be changed to FailIfMissing
```

Comment:
`FailIfMissing` = `ReportIfMissing`? I only see `FailIfMissing` mentioned in the comments.

Reply:
my first iteration of this ReportIfMissing was called FailIfMissing but i changed the name to be more descriptive and forgot to change the comments

Reply:
Ok, then no. We don't want to map to report some because during calls we want to send to only the clients who are in the call.